sdk/Gemfile

```diff
@@ -12,6 +12,7 @@ group :test, :development do
   gem 'opentelemetry-api', path: '../api'
   gem 'opentelemetry-common', path: '../common'
   gem 'opentelemetry-exporter-zipkin', path: '../exporter/zipkin'
+  gem 'opentelemetry-registry', path: '../registry'
```

Comment:
This is to make local development easier in case you want to make changes to both gems at once.

The big disadvantage to this is that we have to know to coordinate releases all at once.